README.md

```diff
@@ -104,6 +104,15 @@ Then run the following:
   - Opera
     - `yarn run dev:opera`
 
+### :heavy_check_mark: Tests
+
+Don't forget to run the tests before committing anything!
+Just run the command below to run all tests:
```

Comment:
Maybe describe previous necessary steps which might be needed.  

Does it make sense to split this up into "Unit tests" and "Integration/e2e tests"?